src/main/resources/db/postgresql/V1_17__CREATE_TABLE_COUNTRY.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS country;
```

Comment:
should this file be included?

Reply:
yes, as discussed it's for flyway consistency when we merge wcs branch back to cs